<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerID</class>
 <widget class="QDialog" name="registerID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>英语资历：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwrdEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EnglishEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入英语资历</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="canclBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>184</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入身份证号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confrmBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>184</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confrmPswrdEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入确认密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rtrnLoginBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>返回登陆界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="phonenumRgstrBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用手机号注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份证号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="usrnameRegisterBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用用户名注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
